api/v1/kubectlstorageosconfig_types.go

```diff
@@ -80,6 +80,8 @@ type Install struct {
 	EtcdEndpoints                   string `json:"etcdEndpoints,omitempty"`
 	EtcdTLSEnabled                  bool   `json:"etcdTLSEnabled,omitempty"`
 	EtcdSecretName                  string `json:"etcdSecretName,omitempty"`
+	EtcdDockerRepository            string `json:"etcdDockerRepository,omitempty"`
```

Comment:
Small detail
In `storageos/node:latest` the repository is "storageos", shouldn't this be named "image" instead?

Reply:
Yeah it's a bit ambiguous, but I used the same terminology the etcd-operator uses (the flag it uses is `--etcd-repository`).

 I'm happy to change it if other agree that it's confusing

Reply:
I would have considered `storageos` to be the organization, and `node` to be the repo.... but maybe my whole life has been a lie 🤯

Reply:
Also, do we need to specify the repository? surely the tag is enough?

Reply:
Yeah this is a requirement from customer success. Some customers like to host their own images